etl/etl.py

```diff
@@ -100,7 +101,29 @@ def unique_insert_edge(g, row, target, label):
         ).iterate()
 
 
-def gremlin_queries(dataframe):
+def ingest_graph(dataframe: pd.DataFrame) -> None:
```

Comment:
Could we update this method to accept the Gremlin server address as an argument instead of hard-coding it inside the function? For now it does not make a big difference, but if / when we need to update the Gremlin server (e.g. change the port) then it will be useful to have this parameterized.

Reply:
Good point! :+1: